conans/client/tools/win.py

```diff
@@ -93,8 +93,27 @@ def latest_vs_version_installed(output):
     return latest_visual_studio_version_installed(output=output)
 
 
+MSVS_DEFAULT_TOOLSETS = {"16": "v141",
```

Comment:
I think 16 should be removed until we have better understanding of VS2019, its representation in settings.yml, etc.

Reply:
I've removed all traces of VS2019 (16), as it has its own PR (https://github.com/conan-io/conan/pull/4233)